codebuild/bin/install_al2_dependencies.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+set -eu
+source ./codebuild/bin/s2n_setup_env.sh
+
+base_packages() {
+    yum update -y
+    yum erase -y openssl-devel || true
+    yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm || true
+    yum install amazon-linux-extras
+
+    # Who owns this package? It needs updating to not install modules under python2.7
+    PYTHON=$(which python2) amazon-linux-extras install -y ruby2.6 rust1 python3.8
+    PYTHON=$(which python2) amazon-linux-extras enable epel
+    PYTHON=$(which python2) amazon-linux-extras enable corretto8
+    yum install -y openssh-clients
+}
+
+mono() {
+    rpm --import https://download.mono-project.com/repo/xamarin.gpg
+    curl https://download.mono-project.com/repo/centos7-stable.repo | tee /etc/yum.repos.d/mono-centos7-stable.repo
+}
+
+symlink_all_the_things() {
+    # Package owners should be doing this.
+    # Note the version number at the end allows for upgrades to superceeded these.
```

Comment:
s/superceeded/superceed/

Reply:
To be especially pedantic, it is spelled 'supersede'

Reply:
Updated; however the online dictionaries don't agree on this point.  supercede, supersede and superseded are valid, depending on where you land.

Reply:
nothing like the English language to inject some ambiguity :)  Personally, I prefer to supercede things, but that's jut the Brit in me...